pandas/core/strings.py

```diff
@@ -413,58 +413,12 @@ def _get_single_group_name(rx):
         return None
 
 
-def str_extract(arr, pat, flags=0):
+def str_extract_original(arr, pat, flags=0):
```

Comment:
maybe call this ``str_extract_data_dependent_expand`` (or if you have a better name, ``original`` doesn't do it for me)